client/app/scripts/superdesk-publish/publish.js

```diff
@@ -205,8 +229,21 @@
             newItem.output_channel_id = item.output_channel_id;
             newItem.unique_name = item.unique_name;
             newItem.destination = item.destination;
+            return newItem;
+        };
+
+        $scope.scheduleToSend = function(item) {
+            var queueItems;
+
+            if (angular.isDefined(item)) {
+                queueItems = $scope.buildNewSchedule(item);
```

Comment:
There are some properties which become invalid in case of post like error_message,...in future there can be some others, that's the reason am building a new object.